spec/game_spec.rb

```diff
@@ -0,0 +1,136 @@
+require 'game'
+
+describe Game do
+  it 'can be initialized' do
+    expect(Game).to respond_to(:new)
+  end
+
+  it 'initializes with starting conditions' do
+    expect(subject.current_turn).to eq 0
+    expect(subject.game_score).to eq 0
+    expect(subject.current_frame).to eq nil
+    expect(subject.frames).to eq []
+  end
+
+  describe '#start_new_frame' do
+    it 'begings the game and proceeds to first frame' do
+      subject.start_new_frame
+      expect(subject.current_turn).to eq 1
+      expect(subject.current_frame).to be_kind_of Frame
+    end
+  end
+
+  describe '#roll_current_frame' do
+    before(:each) do
+      subject.start_new_frame
+    end
+
+    it 'updates the current frame based on rolls' do
+      subject.roll_current_frame(5, 3)
+      expect(subject.current_frame.throws).to eq [5, 3]
+    end
+
+    it 'will work for a strike also' do
+      subject.roll_current_frame(10)
+      expect(subject.current_frame.throws).to eq [10]
+    end
+
+    it 'can call on add_roll_score' do
+      allow(subject.current_frame).to receive(:add_roll_score).with(4)
+      allow(subject.current_frame).to receive(:add_roll_score).with(2)
```

Comment:
This test has no expectation. As such, it could never fail. maybe you wanted to write:
```
expect(subject.current_frame).to receive(:add_roll_score).with(4)
```

allow and expect have different behaviours.

Reply:
For some reason I think I misread this syntax once and have been using allow instead of expect for awhile. Thanks for pointing it out 😬